src/components/NavBar.js

```diff
@@ -64,6 +84,7 @@ class NavBar extends Component {
         username,
         isAdmin: groups && groups[0] === 'Admin',
       });
+      window.location.reload()
```

Comment:
why reload?

Reply:
for instance, if you were logged in, and you log out the like buttons would still be highlighted and vice versa.